tests/Stats/Reader/RedisReaderTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace App\Tests\Stats\Reader;
+
+use App\Stats\Reader\RedisReader;
+use PHPUnit\Framework\TestCase;
+use Predis\Client as Redis;
+
+class RedisReaderTest extends TestCase
+{
+    /**
+     * @dataProvider readsTotalAccess
+     */
+    public function testItReadsTotalAccess(string $prefixKey, string $totalKey, $total, int $expectedTotal): void
+    {
+        $redis = $this->getMockBuilder(Redis::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $redis->method('__call')
+            ->with('get', [\sprintf('%s.%s', $prefixKey, $totalKey)])
+            ->willReturn($total);
+
+        $reader = new RedisReader($redis);
+
+        $this->assertEquals($expectedTotal, $reader->totalAccess());
+    }
+
+    public function readsTotalAccess(): array
+    {
+        return [
+            [
+                'STAT', 'TOTAL', 10, 10,
+            ],
+            [
+                'STAT', 'TOTAL', '10', 10,
+            ],
+            [
+                'STAT', 'TOTAL', null, 0,
+            ],
+            [
+                'STAT', 'TOTAL', 'aString', 0, // @todo is this the expected behaviour?
```

Comment:
@JellyBellyDev please, take a look also at this

Reply:
Yes I see! It's a old behaviour and for retro-compatibility stats, but these statistics are not shown anywhere currently. They were and are being written for future analysis. Maybe someday we'll do something about it :)

Reply:
So it is desired behaviour to convert any possible string to `0`?
If so, I'll remove the comment.

Reply:
Yep!